src/graph.rs

```diff
@@ -531,7 +531,7 @@ pub struct Dependency {
   pub maybe_type: Resolved,
   #[serde(skip_serializing_if = "is_false")]
   pub is_dynamic: bool,
-  #[serde(rename = "assertType", skip_serializing_if = "Option::is_none")]
+  #[serde(rename = "assertionType", skip_serializing_if = "Option::is_none")]
```

Comment:
The types for this are `assertionType` and `serialize_dependencies` has this as `assertionType` so changing this to match that. I think this was only used on the `Module#dependencies` property  in js_graph, but maybe I'm wrong.